READMEs/main-flow.md

```diff
@@ -102,8 +102,25 @@ file_name = os.path.join(asset_dir, "file0")
 Let's check that the file is downloaded. In a new console:
 
 ```console
-cd my_project/datafile.did:op:0xAf07...
-ls branin.arff
+cd my_project/datafile.did:op:*
+vi file0
```

Comment:
can we use `cat` here please?

Reply:
Cat opens the end of the file, however. That's why I went with vi. Is there a reason to prefer cat?

Reply:
maybe some of the users do not have experience with vim and `cat` is used only for listing the content of a specific file. I personally use it often.